man/coredumpctl.xml

```diff
@@ -119,6 +119,16 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-d</option> <replaceable>DEBUGGER</replaceable></term>
```

Comment:
The short option was removed from the code, so please drop it from the manpage as well.